db_provision.mjs

```diff
@@ -68,19 +103,46 @@ function fromSephoraObject(sephoraProduct) {
   };
 }
 
-async function insertProduct(id, brand_id, name, description, ingredients) {
+async function insertCategory(id, name, slug) {
+  try {
+    await client.sql`
+    INSERT INTO Categories (
+      id,
+      name,
+      slug
+    )
+    VALUES (${id}, ${name}, ${slug});
+  `;
+    console.log(`Successfully inserted category ${name}`);
+    return id;
+  } catch (error) {
+    console.error(error);
```

Comment:
Let's also log the category name. In case something goes wrong we can debug it

```js
console.log(`Failed to insert category with id:${id}`, error);
```